packages/backend/src/controllers/bills/bills.util.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@nestjs/common';
+import { BillDocument } from 'src/db/bill/bill.schema';
```

Comment:
I would suggest change all ````src```` to relative path, but as long as it is not creating errors for testing and running then all good